src/azure/cli/application.py

```diff
@@ -175,6 +176,25 @@ def _register_builtin_arguments(**kwargs):
         global_group.add_argument('--debug', dest='_log_verbosity_debug', action='store_true',
                                   help='Increase logging verbosity to show all debug logs.')
 
+    @staticmethod
+    def _expand_file_prefixed_files(argv):
+        return list(
+            [Application._load_file(arg[1:]) if arg.startswith('@') else arg for arg in argv]
+            )
+
+    @staticmethod
+    def _load_file(path):
+        try:
+            if path == '-':
+                content = sys.stdin.read()
+            else:
+                with open(path, 'r') as input_file:
+                    content = input_file.read()
+
+            return content[0:-1] if content[-1] == '\n' else content
```

Comment:
>return content[0:-1] if content[-1] == '\n' else content [](start = 12, length = 56)

why do we strip the final newline?  Also, on Windows this would leave a trailing \r.